packages/dso-toolkit/src/components/tabs/tabs.scss

```diff
@@ -99,8 +99,9 @@
       display: block;
       line-height: typography.$line-height-base;
       margin-inline-end: 2px;
-      min-block-size: css-tabs-variables.$height;
-      padding: css-tabs-variables.$horizontal-padding css-tabs-variables.$vertical-padding;
+      min-block-size: css-tabs-variables.$block-size;
+      padding-block: css-tabs-variables.$block-padding;
```

Comment:
Weet of het precies van deze regel komt, maar moet wel uit deze of de variable file komen, maar ik denk dat de screenshot diff een terechte is:
![html-css-tabs--default diff](https://github.com/dso-toolkit/dso-toolkit/assets/4646174/642099b9-67d9-4f5a-932c-e87513c252a5)
![html-css-tabs--inactief diff](https://github.com/dso-toolkit/dso-toolkit/assets/4646174/8d32c774-79d4-4f1f-87ce-d9afda56a784)

Er is een duidelijk hoogte verschil.

Reply:
Ha! ik zie het al: de auteur. van de oorspronkelijke code heeft verticaal en horizontaal omgekeerd gebruikt, en ik heb dat onbewust 'gecorrigeerd'... Ik zal het ook verkeerd doen hier :)

Reply:
aangepast met knijper op de neus...

Reply:
Oh genieten zeg 🖼️